app/models/concerns/language_and_schedule_references.rb

```diff
@@ -5,6 +5,20 @@ module LanguageAndScheduleReferences
     has_many :language_skills, as: :languageable, dependent: :destroy
     accepts_nested_attributes_for :language_skills, allow_destroy: true
 
+    def first_language
```

Comment:
The name `first_*` implies that the method returns a single item, not a collection.